torch/csrc/jit/operator_upgraders/utils.h

```diff
@@ -30,5 +30,9 @@ TORCH_API bool isOpSymbolCurrent(
     const std::string& name,
     size_t current_version);
 
+TORCH_API std::vector<std::string> loadPossibleHistoricOps(
```

Comment:
Consider adding comment on what this method does.

Reply:
done